pkg/plugin/aggregation/handler.go

```diff
@@ -37,47 +39,74 @@ const (
 	PathResultsByNode = "/api/v1/results/by-node"
 
 	// PathResultsGlobal is the path for global (non-node-specific) results to be PUT to. Callers should
-	// add one path elements as a suffix to this to specify the plugin name (e.g. `<path>/plugin`)
+	// add one path element as a suffix to this to specify the plugin name (e.g. `<path>/plugin`)
 	PathResultsGlobal = "/api/v1/results/global"
 
+	// PathStatusByNode is the path for node-specific status to be POSTed to. Callers should
+	// add two path elements as a suffix to this to specify the node and plugin (e.g. `<path>/node/plugin`)
+	PathStatusByNode = "api/v1/status/by-node"
```

Comment:
I thought that the API path for this would also have been changed. Did you want to keep it as `status`?

Reply:
No lets change that as well.